samples/bluetooth/direction_finding_peripheral/sample.yaml

```diff
@@ -0,0 +1,17 @@
+sample:
+  name: Direction Finding Peripheral
+  description: Sample application showing peripheral role of Direction Finding in connected mode
+tests:
+  sample.bluetooth.direction_finding_peripheral:
+    harness: bluetooth
+    platform_allow: nrf52833dk_nrf52833
+    tags: bluetooth
+    integration_platforms:
+        - nrf52833dk_nrf52833
+  sample.bluetooth.direction_finding_peripheral.aod:
+    extra_args: OVERLAY_CONFIG="overlay-aoa.conf"
+    harness: bluetooth
+    platform_allow: nrf52833dk_nrf52833
```

Comment:
nrf52820dk_nrf52833 and nRF53 DK in both variants ?

Reply:
Will add it in follow up PR.